### ISSUE TITLE
x86/qt-app build error

### ISSUE THREAD

**Issue by sikmir** [embox/embox#1292]
Steps to reproduce:
```bash
$ make confload-x86/qt-app
$ make
...
src/drivers/console/fbcon.c:157:12: error: 'char_dev_idesc_fstat' undeclared here (not in a function)
  .fstat  = char_dev_idesc_fstat,
            ^~~~~~~~~~~~~~~~~~~~
make[4]: *** [mk/image_lib.mk:33: build/base/obj/src/drivers/console/fbcon.o] Error 1
...
```

**Comment by AntonKozlov** [comment]
Do we have the template in CI?

**Comment by Deryugin** [comment]
No, we don't.

**Comment by anton-bondarev** [comment]
Fix it in #1293 Check please

**Comment by sikmir** [comment]
It's buildable now.
